src/popup/router/components/NodeConnectionStatus.vue

```diff
@@ -79,4 +93,28 @@ export default {
     bottom: 0;
   }
 }
+
+.node-connection-status {
+  &.relative {
```

Comment:
```suggestion
.node-connection-status.relative {
```
In such case we don't need to split it into two rows.

Reply:
I don't agree. The `.relative` is a state so it's easier to follow the CSS when this is done like Paweł did it.
But to be honest I think this component should not control its position relative to its parent component. The parent should place it in correct place.